src/main/java/pl/stormit/tinyurl/service/UrlExpiryService.java

```diff
@@ -0,0 +1,54 @@
+package pl.stormit.tinyurl.service;
+
+
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import pl.stormit.tinyurl.domain.model.Url;
+import pl.stormit.tinyurl.domain.model.UrlExpiry;
+import pl.stormit.tinyurl.domain.repository.UrlExpiryRepository;
+
+import java.time.Instant;
+import java.util.List;
+
+@Service
+@AllArgsConstructor
+public class UrlExpiryService {
+
+    @Autowired
+    private final UrlExpiryRepository urlExpiryRepository;
+
+    public UrlExpiry createUrlExpiryDate(Url url) {
+
+        UrlExpiry urlExpiry = new UrlExpiry();
+
+        urlExpiry.setUrl(url);
+
+        return urlExpiryRepository.save(urlExpiry);
+    }
+
+    @Scheduled(fixedDelay = 15000)
+    public void deleteExpiredDateUrls() {
+
+        System.out.println();
+        System.out.println("Start searching expired urls");
```

Comment:
If we want to display information in the LOG, you should use a logger (e.g. slf4j ) instead of "System.out.println()"